apps/reference/docs/guides/platform/logs.mdx

```diff
@@ -4,78 +4,198 @@ title: Logging
+
+The Logs tab displays logs emitted during function execution.
+
+![Function Logs](/img/guides/platform/logs/logs-functions.png)
+
+  </TabItem>
+</Tabs>
+
+## Logs Explorer
+
+The [Logs Explorer](https://app.supabase.com/project/_/logs-explorer) exposes logs from each part of the Supabase stack as a separate table that can be queried and joined using SQL.
+
+![Logs Explorer](/img/guides/platform/logs/logs-explorer.png)
+
+You can access the following logs from the **Sources** drop-down:
+
+- `auth_logs`: GoTrue server logs, containing authentication/authorization activity.
+- `edge_logs`: Edge network logs, containing request and response metadata retrieved from Cloudflare.
+- `function_edge_logs`: Edge network logs for only edge functions, containing network requests and response metadata for each execution.
+- `function_logs`: Function internal logs, containing any `console` logging from within the edge function.
+- `postgres_logs`: Postgres database logs, containing statements executed by connected applications.
+- `realtime_logs`: Realtime server logs, containing client connection information.
+- `storage_logs`: Storage server logs, containing object upload and retrieval information.
 
-## Log Explorer
+## Querying with the Logs Explorer
 
-![Sidebar navigation steps](/img/guides/platform/logs/sidebar-navigation.png)
+The Logs Explorer uses BigQuery and supports all [available SQL functions and operators](https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators).
 
-The log browser can be accessed in the sidebar under **Logs Explorer**. The **Logs Explorer** is for querying and aggregating project logs across products using SQL `SELECT` queries.
+### Timestamp Display and Behavior
 
-### Example
+Each log entry is stored with a `timestamp` as a `TIMESTAMP` data type. Use the appropriate [timestamp function](https://cloud.google.com/bigquery/docs/reference/standard-sql/timestamp_functions#timestamp) to utilize the `timestamp` field in a query.
 
-For example, you may enter the following into the SQL editor to query for each user's IP address:
+Raw top-level timestamp values are rendered as unix microsecond. To render the timestamps in a human-readable format, use the `DATETIME()` function to convert the unix timestamp display into an ISO-8601 timestamp.
 
 ```sql
-SELECT timestamp, h.x_real_ip
-FROM edge_logs
-  LEFT JOIN UNNEST(metadata) as m ON TRUE
-  LEFT JOIN UNNEST(m.request) AS r ON TRUE
-  LEFT JOIN UNNEST(r.headers) AS h ON TRUE
-WHERE h.x_real_ip IS NOT NULL
-```
+-- timestamp column without datetime()
+select timestamp from ....
+--  1664270180000
 
-![SELECT query example](/img/guides/platform/logs/select-query.png)
+-- timestamp column with datetime()
+select datetime(timestamp) from ....
+-- 2022-09-27T09:17:10.439Z
+```
 
-The list of supported product sources can be found under the **Sources** dropdown.
+:::info
+The rendering behaviour as unix microsecond will be removed in the future.
+:::
```

Comment:
Suggest removing this unless it's absolutely necessary to have. Docs should avoid mentioning anything in the future.